README.md

```diff
@@ -40,6 +40,8 @@ Support [Nick Scialli](https://github.com/nas5w) directly on his repo! [Intervie
 
 | Name         | Location     | Roles                | Citizenship/Visa Requirements | Date Posted <br> mm/dd/yyyy |
 | ------------ | ------------ | -------------------- | ----------------------------- | --------------------------- |
+| [Peraton](https://www.peraton.com/) | Herndon, VA | ✅ [Software Engineer (Entry-level)](https://careers-peraton.icims.com/jobs/136975/job?utm_source=indeed_integration&iis=Job+Board&iisn=Indeed&indeed-apply-token=73a2d2b2a8d6d5c0a62696875eaebd669103652d3f0c2cd5445d3e66b1592b0f&iis=HireClix&iisn=Indeed&mobile=false&width=1296&height=500&bga=true&needsRedirect=false&jan1offset=-300&jun1offset=-240) | - | 08/17/2023 | 
```

Comment:
US Citizenship with the ability to obtain and maintain a TS/SCI security clearance and polygraph prior to starting.